app/index.html

```diff
@@ -24,6 +24,8 @@
   <script type="text/javascript" src="js/livebus/livebus.factory.js"></script>
 
   <script type="text/javascript" src="js/shared/shared.module.js"></script>
+  <script type="text/javascript" src="js/shared/ucBackground/ucBackground.module.js"></script>
+  <script type="text/javascript" src="js/shared/ucBackground/ucBackground.directive.js"></script>
```

Comment:
Directives are a bit different because the directive name is used as an element name (e.g. `<ucBackground></ucBackground>` or an attribute name inside an element (e.g. `<div ucBackground/>`). With services, factories and providers file names _can_ be named after them. But even the names in those are not exactly classes. For example the service name is an instance and the "class" (js does not really have classes) is the constructor function. In your directive case the closest to class is the anonymous function which is a factory that creates the directive. But since it is an anonymous function it doesn't have a name :) (The style guide we use actually advises not to use anonymous functions but we can look into refactoring that later after this PR) If you look at the style guide you linked to https://angular.io/styleguide#!#02-08 the blue box has a filename `validate.directive.ts` even though the directive name is `tohValidate`.
So the file and module names should not be tied to the app name prefix convention of the directive name.